static/src/javascripts/projects/commercial/modules/dfp/performance-logging.js

```diff
@@ -49,10 +49,13 @@ define([
 
     function moduleEnd(moduleName) {
         var timerEnd = userTiming.getCurrentTime();
-        var module = performanceLog.modules.find(function(module){
-            return module.name === moduleName;
-        });
-        if (module) {
+
+        var moduleIndex = performanceLog.modules.map(function (module) {
+            return module.name;
+        }).indexOf(moduleName);
```

Comment:
That's why @janua polyfill service is so awesome, you can (almost) forget about browser compat.